examples/auth_example.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/local/bin/python3.8
```

Comment:
Ok, we're almost ready to merge. Once we've improved the situation around the shebang, I have to say it's not correct shebang 😅  Sorry, I didn't want to make even more of a mess before - at the moment we only have two places where it needs to be improved (simpler).

So: https://stackoverflow.com/a/19305076/7172407
> Correct usage for (defaults to version 3.latest)Python 3 scripts is:
> `#!/usr/bin/env python3`

In our case, it will be: `#!/usr/bin/env python3.8` 😉